File: reaction_time.py
import pyautogui as bot
import time
import webbrowser

(screenWidth, screenHeight) = bot.size()

x = int(screenWidth / 3)
y = int(screenHeight / 2)
tries = 0

webbrowser.open_new('https://humanbenchmark.com/tests/reactiontime')

time.sleep(5)
bot.moveTo(x, y)
time.sleep(1)

bot.leftClick()

color = bot.pixel(x, y)
hexColor = '%02x%02x%02x' % color

while tries < 5:
    while hexColor != '4bdb6a':
        color = bot.pixel(x, y)
        hexColor = '%02x%02x%02x' % color
    
    bot.click()
    tries += 1
    hexColor = ''
    time.sleep(1)
    bot.click()

File: reaction_time.js
import open from 'open';
import robot from 'robotjs';

const sleep = (ms) => new Promise((r) => setTimeout(r, ms));

const screensize = robot.getScreenSize();
const width = screensize.width / 3;
const height = screensize.height / 2;
const GREEN = "4bdb6a";

let tries = 0;

async function start() {
  await open("https://humanbenchmark.com/tests/reactiontime");

  await sleep(3000);

  robot.dragMouse(width, height);
  robot.mouseClick("left");

  while (tries < 5) {
    if (robot.getPixelColor(width, height) === GREEN) {
      robot.mouseClick();
      await sleep(1000);
      robot.mouseClick();
      tries++;
    }
  }
}

start();